src/HotPotato.AspNetCore.Middleware/HotPotato.AspNetCore.Middleware.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Would it be possible to keep .NET Standard in here? If it's messing too much with the package versions too much, I'm not against removing it.

Reply:
Yeah, I'm going to revisit this now that I've read this page: https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-7.0

Specifically:

> However, if you want to share code between .NET Framework and any other .NET implementation, such as .NET Core, your library should target .NET Standard 2.0. [No new versions of .NET Standard will be released](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/), but .NET 5, .NET 6, and all future versions will continue to support .NET Standard 2.1 and earlier.

So I think we may want to target netstandard2.1 at least for the main components other folks will be integrating with (Middleware, Core, and OpenAPI plus any supporting libraries). I think the host is fine to keep on net6, but I'm gonna play around with that a bit.